src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -321,9 +412,77 @@ public static void GetCursorPosition()
         }
     }
 
+    [Fact]
+    public void CursorLeft_Set_GetReturnsExpected()
+    {
+        if (!Console.IsInputRedirected && !Console.IsOutputRedirected)
```

Comment:
When running the tests normally, does this block execute?

Reply:
Not sure - don't think so but this follows the other tests in this library, e.g.
```cs

    [Fact]
    [OuterLoop] // clears the screen, not very inner-loop friendly
    public static void Clear_Invoke_Success()
    {
        if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || (!Console.IsInputRedirected && !Console.IsOutputRedirected))
        {
            // Nothing to verify; just run the code.
            Console.Clear();
        }
    }
```